dashboard/app/helpers/aichat_openai_helper.rb

```diff
@@ -39,7 +40,7 @@ def self.format_messages(aichat_model_customizations, stored_messages, new_messa
     )
 
     [
-      {role: "system", content: instructions},
+      {role: "system", content: [{type: "text", text: instructions}]},
```

Comment:
Is this just a more standard input format?

Reply:
Yeah, this is just to make the format consistent for when we parse it below (looking for a key named `content`)